financial-connections/src/main/java/com/stripe/android/financialconnections/features/partnerauth/PartnerAuthViewModel.kt

```diff
@@ -49,6 +52,7 @@ internal class PartnerAuthViewModel @Inject constructor(
     private val completeAuthorizationSession: CompleteAuthorizationSession,
     private val createAuthorizationSession: PostAuthorizationSession,
     private val cancelAuthorizationSession: CancelAuthorizationSession,
+    private val retrieveAuthorizationSession: RetrieveAuthorizationSession,
```

Comment:
@carlosmuvi-stripe  are you thinking of adding:

```
                    self.dataSource.analyticsClient.log(
                        eventName: "click.prepane.continue",
                        parameters: [
                            "requires_native_redirect": authSession.requiresNativeRedirect
                        ],
                        pane: .partnerAuth
                    )
```

Reply:
I guess this should be click.continue based off meeting

Reply:
sure we can include it. What's `requires_native_redirect`? we're not parsing this field on Android.

Reply:
https://github.com/stripe/stripe-android/pull/7125/commits/8fa7686afe82b9287c89c1e49ddba24afe721633